### PR TITLE
DSYS-26: Cards UI + Tests

### PR DESCRIPTION
**Pull request by lucianoratamero**
**Link to task:** https://labcodes.atlassian.net/browse/DSYS-26
**Link to staging app:** https://card-ui--labstorybook-master.netlify.app/

**Details:**

This PR contains all the UI, playground and tests for the Cards.
Please, check all the aspects of the Cards UI on staging :]

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -13,23 +13,23 @@
       }
```
Comment (falconedani) [inline package-lock.json:13]: Shouldn't these changes be added from a different branch?

File: package.json [hunk 2]
```diff
@@ -20,8 +20,8 @@
     "@babel/core": "^7.7.5",
```
Comment (falconedani) [inline package.json:20]: Shouldn't these changes be added from a different branch?

File: yarn.lock [hunk 3]
```diff
@@ -2,94 +2,60 @@
 # yarn lockfile v1
```
Comment (falconedani) [inline yarn.lock:2]: Shouldn't these changes be added from a different branch?

**Review by CavalcanteLucas — COMMENTED** [review]
LGTM

File: labsystem/src/Card/DoubleAction.test.js [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+      )
+      .toJSON();
+    expect(renderedComponent).toMatchSnapshot();
+  });
+
+  it("renders buttons with light skin", async () => {
+    const renderedComponent = renderer
+      .create(
+        <CardContext.Provider
+          value={{ color: "teal", skin: "vivid", cardType: "filled" }}
```
[the remark this answers is not in the document]
Comment (lucianoratamero) [reply labsystem/src/Card/DoubleAction.test.js:175]: Yup. For these, check the corresponding snapshot file :]

File: labsystem/src/Card/LinkAction.test.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+    expect(renderedComponent).toMatchSnapshot();
+  });
+
+  it("calls props.onClick when clicked", async () => {
+    const mockOnClick = jest.fn();
+    const shallowedLinkAction = shallow(
+      <LinkAction text="test" onClick={mockOnClick} />
+    );
+    shallowedLinkAction.find("a").simulate("click");
+    expect(mockOnClick.mock.calls.length).toEqual(1);
```
[the remark this answers is not in the document]
Comment (lucianoratamero) [reply labsystem/src/Card/LinkAction.test.js:21]: Hmm, no, it wouldn't. I'll add those!

File: package-lock.json (hunk 1 above)
Comment (lucianoratamero) [reply package-lock.json:13]: You're right, but we do need to map this on jira. I'll add a task for this.

File: labsystem/src/Card/LinkAction.test.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (lucianoratamero) [reply labsystem/src/Card/LinkAction.test.js:22]: done! merging...

**Pull request merged**